api-admin/src/main/java/bio/terra/pearl/api/admin/service/forms/SurveyExtService.java

```diff
@@ -55,6 +56,9 @@ public Survey create(String portalShortcode, Survey survey, AdminUser adminUser)
     }
     survey.setPortalId(portal.getId());
     survey.setVersion(1);
+    Instant now = Instant.now();
```

Comment:
Same changes in SurveyController.create() works too but having the changes here in Service seemed cleaner and covers any other service calls other than front-end.